actions/AemClient.js

```diff
@@ -86,7 +86,7 @@ class AemClient {
 
       logger.debug(`Updating variation with ${updates.length} fields`);
 
-      await wretch(updateVariationUrl, true /* retryOnFailure */)
+      await wretch(updateVariationUrl, { shouldRetry: true } /* retryOnFailure */)
```

Comment:
How much time are we going to retry? Is this number configurable?

Reply:
> Is this number configurable?

Yes, this behavior is controlled by the Retry add-on and is fully configurable.